package.json

```diff
@@ -8,7 +8,7 @@
     "lint": "next lint",
     "start": "next start",
     "test": "env TZ='UTC' jest",
-    "storybook": "start-storybook -s ./public -p 6006",
+    "storybook": "export NODE_OPTIONS=--openssl-legacy-provider && start-storybook -s ./public -p 6006",
```

Comment:
Are we sure we need this? I'm ok to merge this in but I want to discuss further on a call

Reply:
I was able to get it to run without it.  Seems to be fixed!